src/patterns/data/default/entity_attribute.tsv

```diff
@@ -4271,3 +4271,4 @@ OBA:2045222	basophilic erythroblast count	CL:0000549	basophilic erythroblast	PAT
 OBA:2045223	myeloid leukocyte count	CL:0000766	myeloid leukocyte	PATO:0000070	amount	myeloid white cell count		https://orcid.org/0000-0001-8314-2140
 OBA:2045224	visceral adipose tissue quantity	UBERON:0035818	visceral fat	PATO:0000070	amount			https://orcid.org/0000-0001-8314-2140
 OBA:2045225	blood clotting level	UBERON:0010210	blood clot	PATO:0000070	amount	thrombus formation level		https://orcid.org/0000-0001-8314-2140
+OBA:2050351	breast mass density	UBERON:0000310	breast	PATO:0001019	mass density	breast density	PMID:37345113	https://orcid.org/0000-0001-6757-4744
```

Comment:
```suggestion
OBA:2050351	breast density	UBERON:0000310	breast	PATO:0001019	mass density	breast tissue density	PMID:37345113	https://orcid.org/0000-0001-6757-4744
```
I suggest you to use "breast density" as the OBA label as that term is used in the title of the [paper that is the source of the GWAS data](https://doi.org/10.3390/cancers15102776).
I think the quality [PATO:0002136 radiopacity](https://www.ebi.ac.uk/ols4/ontologies/pato/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FPATO_0002136?lang=en)
might better reflect what they are strictly measuring/observing by mammography.
However, the radiopacity seen on mammographs correlates with mineral density in tissues, which is loosely related to mass density in the physical sense, so using [PATO:0001019 mass density](https://www.ebi.ac.uk/ols4/ontologies/pato/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FPATO_0001019) would be more consistent with the [HP bone mineral density definition](https://www.ebi.ac.uk/ols4/ontologies/hp/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FHP_0004348?lang=en) and [OBA:1000110 bone density](https://www.ebi.ac.uk/ols4/ontologies/oba/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FOBA_1000110).
Let's ignore how similar [mass density of soil](https://www.ebi.ac.uk/ols4/ontologies/envo/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FENVO_09200005) looks semantically to 'breast density' for now. :)